locallib.php

```diff
@@ -182,6 +183,13 @@ function local_bulkenrol_check_user_mails($emailstextfield) {
     return $checkedemails;
 }
 
+/**
+ * Takes input from text area containing a list of e-mail adresses (optionally group names starting with '#').
+ * Returns an array representation of the input.
+ *
+ * @param unknown $emails input value of the text area.
```

Comment:
There is no param type "unknown" - see https://docs.phpdoc.org/guides/types.html. Can you clarify the type here?